server/src/handlers.py

```diff
@@ -1,39 +1,123 @@
+            httponly=True,
+            max_age=3600 * 24 * 7
+        )
+    return response
+
+
+async def login(request: web.Request) -> web.Response:
+    data = await request.json()
+    remember = data.pop('remember', False)
+    user = User(data)
+    try:
+        user.validate()
+    except DataError as e:
+        return web.json_response(e.to_primitive(), status=400)
+    password = user.password
+    user = await db.get_user(request.app['pool'], user.username)
+    if user is None:
+        return web.json_response({'message': "user don't exists"}, status=404)
+    if user.is_password_same(password):
+        response = web.json_response({'message': 'OK'}, status=200)
+        if remember:
+            response.set_cookie(
+                name='token',
+                value=encrypt_jwt(id=user.id, user_name=user.username),
+                httponly=True
+            )
+        else:
+            response.set_cookie(
+                name='token',
+                value=encrypt_jwt(id=user.id, user_name=user.username),
+                httponly=True,
+                max_age=3600 * 24 * 7
+            )
+        return response
+    return web.json_response({'message': 'wrong password'}, status=400)
+
+
+async def logout(request: web.Request) -> web.Response:
+    response = web.json_response({'message': 'OK'})
+    response.del_cookie(name='token')
+    return response
+
+
+@auth_required
+async def create_game(request: web.Request) -> web.Response:
+    game = await db.create_game(request.app['pool'], request.user)
+    return web.json_response(asdict(game), status=201)
+
+
+@auth_required
+async def login_game(request: web.Request) -> web.Response:
+    data = await request.json()
+    game = await db.get_game(request.app['pool'], data['id'])
+    if game is None:
+        return web.json_response({'message': 'game not found'}, status=404)
+    game.set_opponent(request.user)
+    await db.update_game(request.app['pool'], game)
+    return web.json_response(asdict(game), status=200)
+
+
+async def get_game(request: web.Request) -> web.Response:
+    game = await db.get_game(request.app['pool'], int(request.match_info['num']))
+    if game is None:
+        return web.json_response({'message': 'game not found'}, status=404)
+    return web.json_response(asdict(game), status=200)
 
 
-async def get_list_rooms(request: web.Request) -> web.Response:
-    page = int(request.query.get('page', '1'))
-    limit = int(request.query.get('limit', '10'))
-    rooms, total_count = await db.get_rooms(page, limit)
+async def get_games(request: web.Request) -> web.Response:
+    paginator = Paginator()
+    paginator.create_paginator(
+        request.query.get('page'), request.query.get('limit')
+    )
+    rooms = await db.get_game_list(request.app['pool'], paginator.page, paginator.limit)
+    total_rooms = await db.get_total_games(request.app['pool'])
     return web.json_response({
         "rooms": rooms,
         "paginator": {
```

Comment:
implement method toJSON that will return json-parseble dict with page, limit, and total pages
```python
'paginator': paginator.toJSON()
```

And use or only double quotes, or only single-quotes, don't mix it